### PR TITLE
add example for installing epel composer with php74

### PR DESCRIPTION
**Pull request by OyvindLGjesdal**
I had some issues when updating from php73 to php74 with missing packages, and tried adding different packages to resolve `yum install epel` until it worked.

`php74-pecl-zip` seems to solve it.

Error log from composer.
[composer-error.txt](https://github.com/iusrepo/iusrepo.github.io/files/5076844/composer-error.txt)

output of `sudo yum deplist composer`
[composer-deplist.txt](https://github.com/iusrepo/iusrepo.github.io/files/5077026/composer-deplist.txt)

### CODE REVIEW

**Comment by carlwgeorge** [comment]
Thanks for opening this pull request!

File: markdown/faq.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ This can happen if the package you want to install has dependencies that could
 be satisfied by multiple IUS packages.  Yum's dependency resolution is not
 always smart enough to pull in all the correct dependencies.  You can work
 around this by explicitly requesting a few more package names to help the
-transaction resolve successfully.
+transaction resolve successfully.  `yum deplist non-ius-package` shows dependencies including alternatives.
```
Comment (carlwgeorge) [inline markdown/faq.md:121]: This is a good tip that I wasn't even aware of.  Can you change `non-ius-package` to `<package>`?  It works on IUS packages too, and wrapping it in angle brackets would help indicate that the user needs to substitute the name of the package in question.  Also, please wrap it at 79 characters.

File: markdown/faq.md [hunk 2]
```diff
@@ -130,6 +130,7 @@ to help yum resolve the transaction.
 
 - `yum install composer php72u-{cli,common,gd,intl,mbstring,pdo,process}`
 - `yum install composer php73-{cli,common,gd,intl,mbstring,pdo,process}`
+- `yum install composer php74-{cli,common,gd,intl,mbstring,pdo,process,pecl-zip}`
```
Comment (carlwgeorge) [inline markdown/faq.md:133]: With the addition of php74-pecl-list it appears like we can trim this list down some, can you modify it to `php74-{gd,mbstring,pdo,process,pecl-zip}`?

Comment (OyvindLGjesdal) [reply markdown/faq.md]: Added requested changes. Thanks for the work on ius!

**Comment by carlwgeorge** [comment]
The changes are live on the site, thanks for the contribution.

**Pull request merged**